DFC.ServiceTaxonomy.Editor/Recipes/Service Taxonomy.recipe.json

```diff
@@ -775,14 +776,12 @@
                             }
                         },
                         {
-                            "PartName": "TitlePart",
```

Comment:
I think we need TitlePart? STAX uses this title to display content items in Content Management page. And CUI apps uses this property